examples/pm2-typescript/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "nuxt-ts-edge": "latest",
```

Comment:
We're going to make examples use `stable` version instead of `edge` builds. 
See https://github.com/nuxt/nuxt.js/pull/4874
So I suggest switching to `nuxt-ts`